packages/flutter/lib/src/rendering/table.dart

```diff
@@ -468,7 +467,7 @@ class RenderTable extends RenderBox {
   /// determine the intrinsic size of the column.
   Map<int, TableColumnWidth> get columnWidths => Map<int, TableColumnWidth>.unmodifiable(_columnWidths);
   Map<int, TableColumnWidth> _columnWidths;
-  set columnWidths(Map<int, TableColumnWidth> value) {
+  set columnWidths(Map<int, TableColumnWidth>? value) {
```

Comment:
setter shouldn't be nullable, the getter isn't nullable

i would just replace the `??=` line with an assert that value isn't null